### ISSUE TITLE
gorebuild will fail on Windows or other OSes where os.PathSeparator is not '/'.

### ISSUE THREAD

**Issue by dmitshur** [FiloSottile/gorebuild#1]
I don't see a mention in the README that `gorebuild` is not supposed to be supported on Windows, yet it has an issue that would prevent it from working there.

Many places in the code assume that [`os.PathSeparator`](https://godoc.org/os#PathSeparator) is `/`, but that isn't true on Windows and potentially other OSes.

``` Go
ioutil.ReadDir(build.Default.GOPATH + "/bin")

build.Default.GOPATH+"/bin/"+f.Name()

return strings.TrimPrefix(dir, build.Default.GOPATH+"/src/")
```

The correct fix for this is to use [`filepath`](https://godoc.org/path/filepath) package to manipulate system filepaths, rather than concatenating strings. E.g.:

``` Go
ioutil.ReadDir(filepath.Join(build.Default.GOPATH, "bin"))

filepath.Join(build.Default.GOPATH, "bin", f.Name())
```

**Comment by frou** [comment]
This is true, although the more fundamental issue for Windows users is presumably that the code in dwarf.go currently inspects ELF and Mach-O executables only.
